src/core/transpile.jl

```diff
@@ -1692,10 +1692,6 @@ function transpile(::CompressRzGatesTranspiler, circuit::QuantumCircuit)::Quantu
     )
 end
 
-struct TrivialTranspiler <: Transpiler end
```

Comment:
Why are we removing this?

Reply:
was only used as a dummy transpiler when is_native_circuit was part of `transpile_and_run_job()`. Since I moved that check to a proper transpilation stage, there was no more need for it